docs/apache-airflow/core-concepts/params.rst

```diff
@@ -324,9 +325,18 @@ For examples also please take a look to two example DAGs provided: ``example_par
 .. image:: ../img/trigger-dag-tutorial-form.png
 
 .. versionadded:: 2.7.0
-
-The trigger form can also be forced to be displayed also if no params are defined using the configuration switch
-``webserver.show_trigger_form_if_no_params``.
+    The trigger form can also be forced to be displayed also if no params are defined using the configuration switch
+    ``webserver.show_trigger_form_if_no_params``.
+
+.. versionchanged:: 2.8.0
+    Per default custom HTML is not allowed to prevent injection of scripts or other malicious HTML code. If you trust your DAG authors
```

Comment:
```suggestion
    By default custom HTML is not allowed to prevent injection of scripts or other malicious HTML code. If you trust your DAG authors
```